### ISSUE TITLE
range of time series length

### ISSUE THREAD

**Issue by jimmylao** [Futurewei-io/blue-marlin#223]
The model is seq2seq model. The 1st 'seq' is encoder; the 2nd 'seq' is decoder. Underlying structure of encoder/decoder is GRU/RNN.

The length of decoder is usually determined by use case requirement, for example, if the model is expected to predict 40 days time sequence data, the length of decode equals to 40.

GRU/RNN can usually memorize 100 ~ 300 items. On longer time series data GRU/RNN still works, but can gradually 'forget' information from the oldest items. Attention as well as important (year, halfyear, quarter ago) datapoints from the past are used as additional features for encoder/decoder. In order to get 'year ago' feature, the length of encoder + decoder should be at least one year long (365 days).

In summary, the minimum length of data (encoder + decode, 40 days prediction, for example) is 100 + 40 = 140, the optimal length of data is 365 + 1 ~ 40 = 366 ~ 405.
